slsDetectorServers/slsDetectorServer/src/programViaBlackfin.c

```diff
@@ -29,8 +32,11 @@
 #define CMD_GPIO9_DEFINE_IN "echo in > /sys/class/gpio/gpio9/direction"
 #define CMD_GPIO3_DEFINE_IN "echo in > /sys/class/gpio/gpio3/direction"
 
+// nConfig
 #define CMD_GPIO9_DONT_TOUCH_FLASH "echo 0 > /sys/class/gpio/gpio9/value"
-#define CMD_GPIO3_DONT_TOUCH_FLASH "echo 0 > /sys/class/gpio/gpio3/value"
+// nCE
+#define CMD_GPIO3_DONT_TOUCH_FLASH "echo 1 > /sys/class/gpio/gpio3/value"
```

Comment:
Hi @thattil 

I am interested in understanding the origin of this change. In particular, I would like to know if it is exclusive of Jungfrau FW v2_3, released by @lopez-c on April, or if it applies as well to v2_2.